README.rst

```diff
@@ -50,15 +50,6 @@ It checks the following locations:
 - Python files, ``.txt`` and ``.rst`` files for imports in doctests.
 
 
-Note on running the tests
```

Comment:
Isn't this paragraph still valid for local development?

Reply:
Well, tests are no longer that sensitive, at least that's my aim 🙂 

With the current travis setup, that one should replicate for local development, the environment is already isolated.

Reply:
The you should point at the `.travis.yml` file and say that you should replicate that for local development.

I think it is easier to just keep the current text (until the tests are actually less sensitive).

Reply:
ok